docs/source/tutorials/multiannotator.ipynb

```diff
@@ -139,7 +144,7 @@
    "id": "345b6678",
    "metadata": {},
    "source": [
-    "## 2. Create the data (can skip these details)"
+    "## 2. Format data (can skip these details)"
```

Comment:
```suggestion
    "## 2. Create the data (can skip these details)"
```

I think for this tutorial we can keep it as "creating data" bc we are creating the synthetic data